docs/content/core/logger.mdx

```diff
@@ -406,3 +406,23 @@ def test_lambda_handler(lambda_handler, lambda_context):
     test_event = {'test': 'event'}
     lambda_handler(test_event, lambda_context) # this will now have a Context object populated
 ```
+
+## FAQ
+
+**Why don't I see my library logs?**
```

Comment:
That's not true anymore since 1.7.0, or 9 days ago to be more precise :)

Reply:
Ah right... I totally missed that :-) Seen it working now that I actually tried enabling the `boto3` and `botocore` logs myself ;-)